packages/flutter/lib/src/services/message_codec.dart

```diff
@@ -118,8 +119,11 @@ class PlatformException implements Exception {
   /// Error details, possibly null.
   final dynamic details;
 
+  /// Native stacktrace for the error, possibly null.
```

Comment:
nits:
Can we explain more?
Like what's "Native stacktrace", where does user get dart stack trance, etc.